src/renderers/shared/fiber/ReactChildFiber.js

```diff
@@ -219,3 +221,54 @@ function ChildReconciler(shouldClone) {
 exports.reconcileChildFibers = ChildReconciler(true);
 
 exports.reconcileChildFibersInPlace = ChildReconciler(false);
+
+
+function cloneSiblings(current : Fiber, workInProgress : Fiber, returnFiber : Fiber) {
+  workInProgress.return = returnFiber;
+  while (current.sibling) {
+    current = current.sibling;
+    workInProgress = workInProgress.sibling = cloneFiber(
+      current,
+      current.pendingWorkPriority
+    );
+    workInProgress.return = returnFiber;
+  }
+  workInProgress.sibling = null;
+}
+
+exports.cloneChildFibers = function(workInProgress : Fiber) {
+  if (!workInProgress.child) {
+    return;
+  }
+  const current = workInProgress.alternate;
```

Comment:
Normally I pass these in. I should just pass it in as an argument instead of relying on the alternate field.